UltraStar Play/Assets/Common/Video/SongVideoPlayer/SongVideoPlayer.cs

```diff
@@ -260,14 +278,25 @@ private void InitVideo(SongMeta songMeta)
         }
     }
 
+    void OnEnable()
+    {
+        videoPlayer.errorReceived += OnVideoPlayerErrorReceived;
+    }
+
     void OnDisable()
     {
+        videoPlayer.errorReceived -= OnVideoPlayerErrorReceived;
         if (HasLoadedVideo)
         {
             ClearOutRenderTexture(videoPlayer.targetTexture);
         }
     }
 
+    private void OnVideoPlayerErrorReceived(VideoPlayer source, string message)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'source'.](https://app.codacy.com/manual/UltraStar-Deluxe/Play/pullRequest?prid=5520139)